shared/authplugin/src/main/java/io/pravega/auth/AuthConstants.java

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.common.auth;
+package io.pravega.auth;
```

Comment:
This is another change unrelated to this PR. Let's not interleave formatting changes in PRs that are not refactoring the given code. In fact, I want to understand why we are moving this class and others in this PR from `common` to `shared`. Is there even an issue for that and has it been discussed?

Reply:
The interface `AuthHandler` had changed as a response to a comment on the PR. Which mandated changing of the location of the exceptions in this package. As with the current changes, they needed to be visible to the plugin implementation. The interface change was discussed and the move of the exceptions to new packages is implicit as part of the interface change.
Once the exceptions are moved, then there is only class left in this package and I moved that too. Do let me know whether this justifies the move for this particular class.